### PR TITLE
Semantic Scholar integration Allows semantic scholar integration with Elastic Search

### PR DESCRIPTION
**Pull request by ArthurCamara**

### CODE REVIEW

File: app/services/search/providers/elasticsearch.js [hunk 1]
```diff
@@ -22,9 +23,19 @@ exports.fetch = function (query, vertical, pageNumber, resultsPerPage, relevance
     }
     if (vertical in verticals) {
         const dataset = verticals[vertical];
+        console.log(JSON.stringify(dataset.custom_query(query)))
```
Comment (felipemoraes) [inline app/services/search/providers/elasticsearch.js:26]: remove log message

File: app/services/search/providers/elasticsearch.js [hunk 2]
```diff
@@ -22,9 +23,19 @@ exports.fetch = function (query, vertical, pageNumber, resultsPerPage, relevance
...
+                from: (pageNumber - 1) * resultsPerPage,
+                size: resultsPerPage,
+                body: dataset.custom_query(query),
+                pretty: true
+            }).then(formatResults(vertical));
+        }
         return esClient.search({
             index: dataset.index,
-            type: 'document',
+            // type: 'document',
```
Comment (felipemoraes) [inline app/services/search/providers/elasticsearch.js:38]: delete or remove type

**Review by felipemoraes — CHANGES_REQUESTED** [review]

File: app/services/search/providers/elasticsearch.js (hunk 2 above)
Comment (ArthurCamara) [reply app/services/search/providers/elasticsearch.js:37]: Solved on 9d96eddedc89ee52e1fe18c09feee80e5f45be81

**Review by ArthurCamara — COMMENTED** [review]
solved on 9d96eddedc89ee52e1fe18c09feee80e5f45be81

**Review by felipemoraes — APPROVED** [review]

**Pull request merged**